api.yml

```diff
@@ -19,9 +19,97 @@ paths:
+    "jws": "eyJhbGciOiJFZERTQSIsImI2..hS5SPmWTFocjcDg",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:example:c6f1c276e12ec21ebfeb1f712eb#jf893k"
+  }]
+}
+                options:
+                  type: object
+                  additionalProperties: false
+                  description: Options for specifying how the credential will be issued.
+                  properties:
+                    checks:
+                      type: array
+                      items:
+                        type: string
+                      description: An array of verification checks to be performed on the credential. WARNING - Implementers are cautioned that the list of checks is currently incomplete and other checks such as issuance, expiration, nonce, domain, and acceptable issuers are expected to be added in time. A default list is also expected to encapsulate all available checks. This option might be changed to a 'ignore' option where it lists the checks the developer would like to skip.
+                      example: ["proof"]
+      responses:
+        '200':
+          description: The verification was successful.
+          content:
+            application/json:
+              schema:
+                type: object
+                description: A description of the checks that were performed.
+                properties:
+                  checks:
+                    type: array
+                    items:
+                      type: string
+                    description: An array of verification checks that were performed. WARNING - Implementers are cautioned that the list of checks is currently incomplete and other checks such as issuance, expiration, nonce, domain, and acceptable issuers are expected to be added in time.
+                    example: ["proof"]
+        '400':
+          description: The verification failed.
+          content:
+            application/json:
+              schema:
+                type: object
+                description: A description of the checks that were performed and any associated errors.
+                properties:
+                  checks:
+                    type: array
+                    items:
+                      type: object
+                    description: An array of verification checks that were performed and any associated errors and other information that may be helpful when debugging the failure. WARNING - Implementers are cautioned that the list of checks is currently incomplete and other checks such as issuance, expiration, nonce, domain, and acceptable issuers are expected to be added in time.
+                    example: [{
+                      "check": "proof",
+                      "error": "The verificationMethod (public key) associated with the digital signature could not be retrieved due to a network error.",
+                      "verificationMethod": "did:example:c6f1c276e12ec21ebfeb1f712eb#jf893k",
+                    }]
+    put:
```

Comment:
Why put??

Reply:
I was trying to preserve the old mechanism since my attempt at redefining what was already there backfired in the Simple Issuer HTTP API.

@peacekeeper wrote:
> I'd be fine with simply replacing the current API with the proposed one here instead of keeping them both, since it only expands the inputs and outputs a bit, which I agree is useful.

Since have noted that you're fine w/ removing the current version in place of this proposed version, I'm happy to do that as that's what I'd prefer as well.

If we do that, what happens to the ".../verifications/{verificationId}" mechanism... do we keep the other endpoint? Or get rid of it?

Reply:
I think you originally proposed the `/verifications/{verificationReference}` GET operation.. I'd be fine with removing it.

Reply:
> I think you originally proposed the /verifications/{verificationReference} GET operation.

Hmm, I wonder why... :) -- in any case, I'll put in a separate PR to remove it if this PR is merged.